### ISSUE TITLE
Use lorem ipsum for screenshots, add alt text

### ISSUE THREAD

**Issue by Tmpod** [LemmyNet/joinlemmy-site#63]
It would be nice to apply i18n to page media, such as images and animations.

**Comment by dessalines** [comment]
What does this mean? Like OCR reading images and memes then translating? that'd pry be way out of scope.

**Comment by dessalines** [comment]
Stale, but I can re-open if someone wants to work on this.

**Comment by Tmpod** [comment]
Completely forgot about this :sweat_smile:  

How could we do this? I don't believe weblate allows you to submit media as a translation.

**Comment by dessalines** [comment]
I'm guessing this is about adding alt tags for images?

You'd need to go over every image on http://jointest.lemmy.ml:1234/ , then add translation strings to joinlemmy-translations repo . 

That's quite a lot of work for little gain, considering most of the media already has descriptions / captions.

**Comment by Tmpod** [comment]
Yes, but not just that. The idea was to actually make new screenshots that show the interface in the selected language.

**Comment by dessalines** [comment]
Yeah. 

I also don't think its a good idea to add screenshots for every language, there are 40+ and that'd be a ton of screenshots to add.

**Comment by Nutomic** [comment]
Yes not really worth the effort. Would be better to create some screenshots with lorem ipsum that dont show any specific language.
